### PR TITLE
Url updates

### PR DESCRIPTION
**Pull request by spadger1066**
Added CreateShipmentRequest and CreateShipmentResponse types to raml
Update to the template to allow multiple types on Request body to facilitate CreateShipmentRequest

### CODE REVIEW

File: _includes/raml_type.html [hunk 1]
```diff
@@ -28,6 +28,7 @@ <h4 class="anchored" id="type-{{ typeDefinition.name | slugify }}">{{ typeDefini
             <th>Type</th>
             <th>Description</th>
           </thead>
+          <tr><td>{{ include.type }}</td><td><td></td></tr>
```
Comment (endremm) [inline _includes/raml_type.html:31]: Double start td tags.

File: _includes/raml_type_row.html [hunk 2]
```diff
@@ -73,6 +73,7 @@
   </td>
 </tr>
 
+<!-- propType is never set to 'object' -->
```
Comment (endremm) [inline _includes/raml_type_row.html:76]: If so, should we fix the code?

Comment (spadger1066) [reply _includes/raml_type_row.html:76]: Yes, there are a number of fixes but I'd like to discuss with you first.

File: _includes/raml_type.html (hunk 1 above)
Comment (spadger1066) [reply _includes/raml_type.html:31]: Column one only. Neater ways of doing it but it's still a work in progress. This commit is to allow others to review what's been done without my having to share my screen all the time. Will discuss on Monday.

**Pull request merged**